samples/javascript_typescript/02.echobot-with-counter/src/bot.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// bot.js is your bot's main entry point to handle incoming activities.
+
+import { ActivityTypes, ConversationState, StatePropertyAccessor } from 'botbuilder';
+
+// Turn counter property
+const TURN_COUNTER = 'turnCounterProperty';
+
+export class EchoBot {
+    /**
+     * 
+     * @param {ConversationState} conversation state object
```

Comment:
This docstring needs to be right on top of the constructor, and the param name (`constructor`) is mismatched with `conversationState`.